src/style.css

```diff
@@ -1 +1,8 @@
 /* styles go here */
+.div__configuration {
```

Comment:
I think this name should describe more what we are trying to style here. It's not really just div, we are styling something that it wraps. Also we know that it is some kind of configuration already :) Something like `form_field` maybe?